salt/modules/win_lgpo.py

```diff
@@ -2442,7 +2442,7 @@ def _binary_enable_zero_disable_one_conversion(cls, val, **kwargs):
                 elif ord(val) == 1:
                     return 'Enabled'
                 else:
-                    return 'Invalid Value'
+                    return 'Invalid Value: %s' % repr(val)
```

Comment:
pylint won't like the percent formatting here. Also, if the string is unicode, repr will include the `u` before the string, is that what we really want?

What is it that we're trying to do here?

Reply:
Chatted with @twangboy on this... the concern is that an invalid value is likely to be a bytestring, such as `\x80`, and we'd like to preserve the bytestring representation because it matches the hex value used in the registry. Instead of using `%` replacement though, we should be using the `!r` format flag and switching to a `.format()`. @twangboy is in the process of making this adjustment.

Reply:
Changed to `.format`

Reply:
It's still a percent substitution, I don't think you updated it.

Reply:
Oh, disregard... for some reason even though I refreshed the browser, when I clicked on "Files changed" it still had the old value there. After I submitted my review it magically updated.